contivctl/commands.go

```diff
@@ -0,0 +1,191 @@
+package contivctl
+
+import (
+	"github.com/codegangsta/cli"
+)
+
+var tenantFlag = cli.StringFlag{
+	Name:  "tenant, t",
+	Value: "default",
+	Usage: "Name of the tenant to create the policy on.",
+}
+
+var Commands = []cli.Command{
+	{
+		Name:  "group",
+		Usage: "Endpoint Group manipulation tools",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
```

Comment:
@erikh, @unclejack @mapur: do you see the first verb being 'object' and verbs/actions as second subcommand a right model given other things in container ecosystem. I'd think so... but wanted to confirm